frontend/coalesce/src/css/app.sass

```diff
@@ -1 +1,13 @@
 // app global css in Sass form
+
+.opportunity-card
+  width: 100%
+
+.body
+  background-color: #dddddd
```

Comment:
Would you mind keeping our primary branding colors in our [quasar.variables.sass](https://github.com/FederationOfTech/Coalesce/blob/master/frontend/coalesce/src/css/quasar.variables.sass)? That way we can easily change them across the site as we break out our sass files later on :)